cpp11/Ice/interceptor/InterceptorI.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#include <InterceptorI.h>
+
+using namespace std;
+
+InterceptorI::InterceptorI(shared_ptr<Ice::Object> servant, shared_ptr<AuthenticatorI> authenticator,
+                           unordered_set<string> securedOperations) :
```

Comment:
I would use a more efficient ordered set... while there isn't many operations for this demo, a customer that take example on it might have a lot more operations.